packages/mux-video/src/CustomVideoElement.js

```diff
@@ -72,9 +70,10 @@ class CustomVideoElement extends HTMLElement {
         if (mutation.type === "childList") {
           // Child being removed
           mutation.removedNodes.forEach((node) => {
-            this.nativeEl.removeChild(
-              this.nativeEl.querySelector(`track[src="${node.src}"]`)
+            const track = this.nativeEl.querySelector(
+              `track[src="${node.src}"]`
             );
+            if (track) this.nativeEl.removeChild(track);
```

Comment:
👍 we'll probably need to account for this overly-presumptuous code better, but the if seems reasonable, given the presumptuousness.